### DISCUSSION TITLE
There is no 'keep-all' in word-break values

### DISCUSSION THREAD

**Discussion by Dann1y** [tailwindlabs/tailwindcss#9444] [Help] [answered]
### word-break utilities
https://tailwindcss.com/docs/word-break

### Mozila word-break
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

I didn't find about `keep-all` values
Is there any alternative for word-break `keep-all`?

### old discussion
https://github.com/tailwindlabs/tailwindcss/discussions/6273

I found a discussion for `keep-all` but I didn't find the solution

**Comment by wongjn** [comment, marked as answer]
You could use an [arbitrary property class](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) for your use case, `[word-break:keep-all]`.

**Reply by Dann1y** [reply]
Yeah I decided to use `addUtilities` in the tailwind.config.js plugin.
Thanks for your advice! 👍
